cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -114,8 +117,12 @@ public void storeOutlet(Outlet outlet, boolean failIfExists) {
     public void deleteOutlet(String officeId, String locationId, DeleteRule deleteRule) {
         connection(dsl, conn -> {
             setOffice(conn, officeId);
-            CWMS_OUTLET_PACKAGE.call_DELETE_OUTLET(DSL.using(conn).configuration(), locationId, deleteRule.getRule(),
-                                                   officeId);
+            try {
+                CWMS_OUTLET_PACKAGE.call_DELETE_OUTLET(DSL.using(conn).configuration(), locationId,
+                        deleteRule.getRule(), officeId);
+            } catch (IntegrityConstraintViolationException e) {
+                throw new NotFoundException(e);
```

Comment:
Not found doesn't make sense given the parent exception. `409 Conflict`  https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409 makes more sense.

Reply:
Swapped for a DeleteConflictException. The error code received by HCDA isn't the most useful, ("Data already exists for this request"). Maybe there's another exception that will provide more detail?

Reply:
It should send basically what the constraint is "something depends on it." We'll have to look into that later.